telethon/events/callbackquery.py

```diff
@@ -26,9 +26,13 @@ class CallbackQuery(EventBuilder):
             can use ``re.compile(b'data_')``.
     """
     def __init__(
-            self, chats=None, *, blacklist_chats=False, func=None, data=None):
+            self, chats=None, *, blacklist_chats=False, func=None, data=None, pattern=None):
```

Comment:
Oh no, you added this into the same pull request. You should have made a separate branch for this change.

Reply:
should be done.